content/en/docs/concepts/overview/kubernetes-api.md

```diff
@@ -71,22 +71,22 @@ separate endpoint for each group version.
 
 ### Aggregated discovery
```

Comment:
Do we need the heading “Aggregated discovery” twice in this one file? If we do, please (briefly) outline why.

If one of those is for OpenAPI, let's add the term “OpenAPI” into the text soon after the heading, so it's extra obvious. I wasn't able to work it out.